K8s/easy-configmap.yml

```
@@ -0,0 +1,15 @@
+  name: easyshop-config
+  namespace: easyshop-hack
+  labels:
+    name: easyshop-config
+    environment: production
+data:
+  MONGODB_URI: "mongodb://mongodb-service:27017/easyshop"
+  NODE_ENV: "production"
+  NEXT_PUBLIC_API_URL: "https://easyshop.letsdeployit.com/api"
+  NEXTAUTH_URL: "https://easyshop.letsdeployit.com/"
+  NEXTAUTH_SECRET: "NEXTAUTH_SECRET"
+  JWT_SECRET: "JWT_SECRET"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)